### PR TITLE
Xian152 country specific 1

### PR DESCRIPTION
**Pull request by Xian152**
country-specific code for 7.do,13.do and 16.do
fix 1.do

### CODE REVIEW

**Review by wengxyu1030 — APPROVED** [review]

File: 7_child_vaccination.do [hunk 1]
```diff
@@ -1,83 +1,215 @@
...
 ******************************
 *** Child vaccination ********
 ******************************   
-
 *c_measles	child			Child received measles1/MMR1 vaccination
-        gen c_measles  =. 
-		replace c_measles = 1 if (h9 ==1 | h9 ==2 | h9 ==3)  
-     	replace c_measles = 0 if h9 ==0  				
-	
+		gen c_measles  = inlist(h9,1,2,3)   if !inlist(h9,8,9,.)
```
Comment (wengxyu1030) [inline 7_child_vaccination.do:5]: This is not as general as the original code, but if you have reviewed all the surveys and sure that only 8,9 are missing it's fine. However, if there are cases where 99 is coded as missing, this command can not capture it as missing instead would code as 0.

File: 7_child_vaccination.do [hunk 2]
```diff
@@ -1,83 +1,215 @@
...
-
 *c_measles	child			Child received measles1/MMR1 vaccination
-        gen c_measles  =. 
-		replace c_measles = 1 if (h9 ==1 | h9 ==2 | h9 ==3)  
-     	replace c_measles = 0 if h9 ==0  				
-	
+		gen c_measles  = inlist(h9,1,2,3)   if !inlist(h9,8,9,.)  
+
+		
+		gen c_polio0 = inlist(h0,1,2,3)  if !inlist(h0,8,9,.)
```
Comment (wengxyu1030) [inline 7_child_vaccination.do:8]: Can you put polio under the polio section? ( for example under *c_polio0)

File: 7_child_vaccination.do [hunk 3]
```diff
@@ -1,83 +1,215 @@
...
+		gen c_polio2 = inlist(h6,1,2,3)  if !inlist(h6,8,9,.) 
+		
+*c_polio3	child			Child received polio3/OPV3 vaccination				
+		gen c_polio3 = inlist(h8,1,2,3)  if !inlist(h8,8,9,.) 
+		
 *c_dpt1	child	Child received DPT1/Pentavalent 1 vaccination	
-        gen c_dpt1  = . 
-		replace c_dpt1  = 1 if (h3 ==1 | h3 ==2 | h3 ==3)  
-		replace c_dpt1  = 0 if h3 ==0  
+		gen c_dpt1  = inlist(h3,1,2,3)  if !inlist(h3,8,9,.)
```
Comment (wengxyu1030) [inline 7_child_vaccination.do:19]: Same issue as discussed for c_measles.

File: 7_child_vaccination.do [hunk 4]
```diff
@@ -1,83 +1,215 @@
...
-        gen c_dpt1  = . 
-		replace c_dpt1  = 1 if (h3 ==1 | h3 ==2 | h3 ==3)  
-		replace c_dpt1  = 0 if h3 ==0  
+		gen c_dpt1  = inlist(h3,1,2,3)  if !inlist(h3,8,9,.)  
+		
 *c_dpt2	child			Child received DPT2/Pentavalent2 vaccination				
-		gen c_dpt2  = . 
-		replace c_dpt2  = 1 if (h5 ==1 | h5 ==2 | h5 ==3)  
-		replace c_dpt2  = 0 if h5 ==0  
+		gen c_dpt2  = inlist(h5,1,2,3)  if !inlist(h5,8,9,.)
```
Comment (wengxyu1030) [inline 7_child_vaccination.do:22]: the same issue, please elaborate.

**Review by wengxyu1030 — APPROVED** [review]

File: 13_adult.do [hunk 5]
```diff
@@ -32,18 +59,23 @@ capture confirm variable sh246s sh255s sh264s sh246d sh255d sh264d
...
 	}
 	if _rc != 0{
 	gen a_bp_dial = .
 	}
+
 	
 *a_hi_bp140_or_on_med	18y+ with high blood pressure or on treatment for high blood pressure	
 	gen a_hi_bp140=.
-    replace a_hi_bp140=1 if a_bp_sys>=140 | a_bp_dial>=90 
+    replace a_hi_bp140=1 if (a_bp_sys>=140 & a_bp_sys!=.) | (a_bp_dial>=90 & a_bp_dial!=.)
```
Comment (wengxyu1030) [inline 13_adult.do:78]: Thanks a lot, this is very helpful, I will review the other rounds to make the changes accordingly.

File: 7_child_vaccination.do (hunk 1 above)
Comment (Xian152) [reply 7_child_vaccination.do:5]: Yeah, you are right. I did check all points and found it is ok with VI. But to make sure it applies to all versions, I will change it back.

File: 7_child_vaccination.do (hunk 2 above)
Comment (Xian152) [reply 7_child_vaccination.do:8]: sure

**Pull request closed without merging**